extensions/import/src/wizard/pages/summaryPage.ts

```diff
@@ -0,0 +1,165 @@
+	private loading: sqlops.LoadingComponent;
+	private form: sqlops.FormContainer;
+
+	public constructor(instance: FlatFileWizard, model: ImportDataModel, view: sqlops.ModelView, provider: FlatFileProvider) {
+		super(instance, model, view, provider);
+	}
+
+	async start(): Promise<boolean> {
+		this.table = this.view.modelBuilder.table().component();
+		this.statusText = this.view.modelBuilder.text().component();
+		this.loading = this.view.modelBuilder.loadingComponent().withItem(this.statusText).component();
+
+		this.form = this.view.modelBuilder.formContainer().withFormItems(
+			[
+				{
+					component: this.table,
+					title: localize('flatFileImport.importInformation', 'Import information')
+				},
+				{
+					component: this.loading,
+					title: localize('flatFileImport.importStatus', 'Import status')
+				}
+			]
+		).component();
+
+		await this.view.initializeModel(this.form);
+		return true;
+	}
+
+	async onPageEnter(): Promise<boolean> {
+		this.loading.loading = true;
+		this.populateTable();
+		await this.handleImport();
+		this.loading.loading = false;
+		this.instance.setImportAnotherFileVisibility(true);
+
+		return true;
+	}
+
+	async onPageLeave(): Promise<boolean> {
+		this.instance.setImportAnotherFileVisibility(false);
+
+		return true;
+	}
+
+	private populateTable() {
+		this.table.updateProperties({
+			data: [
+				['Server name', this.model.server.providerName],
+				['Database name', this.model.database],
+				['Table name', this.model.table],
+				['Table schema', this.model.schema],
+				['File to be imported', this.model.filePath]],
+			columns: ['Object type', 'Name'],
+			width: 600,
+			height: 200
+		});
+	}
+
+	private async handleImport(): Promise<boolean> {
+		let changeColumnResults = [];
+		this.model.proseColumns.forEach((val, i, arr) => {
+			let columnChangeParams = {
+				index: i,
+				newName: val.columnName,
+				newDataType: val.dataType,
+				newNullable: val.nullable,
+				newInPrimaryKey: val.primaryKey
+			};
+			changeColumnResults.push(this.provider.sendChangeColumnSettingsRequest(columnChangeParams));
+		});
+
+		let result: InsertDataResponse;
+		let err;
+		try {
+			result = await this.provider.sendInsertDataRequest({
+				connectionString: await this.getConnectionString(),
+				//TODO check what SSMS uses as batch size
+				batchSize: 500
+			});
+		} catch (e) {
+			err = e.toString();
+		}
+
+		let updateText: string;
+		if (!result || !result.result.success) {
+			updateText = '✗ ';
+			if (!result) {
+				updateText += err;
+			} else {
+				updateText += result.result.errorMessage;
+			}
+		} else {
+			let rows = await this.getCountRowsInserted();
+			if (rows < 0) {
+				updateText = localize('flatFileImport.success.norows', '✔ Awesome! You have successfully inserted the data into a table.');
+			} else {
+				updateText = localize('flatFileImport.success.rows', '✔ Awesome! You have successfully inserted %rows% rows.').replace('%rows%', String(rows));
```

Comment:
String formatting like this doesn't work with localization. See the docs at https://github.com/Microsoft/vscode-nls for an example of the best way to do it